app/enterprise/0.33-x/developer-portal/best-practice.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Kong Developer Portal Best Practices
+book: portal
+chapter: 12
+---
+
+# Kong Developer Portal Best Practices
+
+The Kong Developer Portal is a powerful tool. It's part static website and part interactive API. With  tool of this magnitude, there are countless ways to configure and host that will vary greatly depending on your needs. The following article is a group of recommendations we have found to lead which lead to  easy administrative and top performance. This list is not all inclusive, so if you have a trick you like to use and want to share it with others, please feel free to make a PR on the page, or send us an email at <support@konghq.com> and we can add it for you.
+
+___
+
+- [Basic System Requirements and Initial Configuration](#systemConfig)
+- [Customizing The Portal Design with CSS](#css)
+- [Customizing Front End Behavior with JavaScript](#js)
+- [Using Source Control](#git)
+- [Effective Partial Usage](#partial)
+- [Practices to Avoid](#avoid)
+___
+
+## <a name=systemConfig></a>Basic System Requirements and Initial Configuration
+
+When deciding what type of system to host Kong on, you should take into consideration into the amount of traffic you expect the portal to see. If you are using your portal only for internal APIs shared between 2 teams, you'll need a much smaller instance than someone with a public facing, heavily trafficked API. By default, I suggest starting with a more powerful machine than you need and scaling back. if you have a bit more power than you need, none of your end users will notice a difference. However, if you start too small, it could potentially lead to a slower performance of the portal. This section is going to assume your portal gets about 1000 concurrent visitors on average.
```

Comment:
The requirements of Kong are not the same as the requirements for the Dev portal. People may start at the same instance size, but different things need to be considered.

Reply:
I can buy that! Can you at least cross link between the two docs, so that when folks read this doc it says something like "This other document has more general Kong requirements" (with a link of course) and, in that other doc, add a cross link "See this other doc for Kong Dev Portal requirements"